dp/cloud/go/services/dp/logs_pusher/logs_pusher_test.go

```diff
@@ -0,0 +1,53 @@
+package logs_pusher
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type stubFluentdServer struct {
+	expectedPayload string
+	response        string
+	t               *testing.T
+}
+
+func (s *stubFluentdServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	p, _ := ioutil.ReadAll(r.Body)
+	assert.JSONEq(s.t, s.expectedPayload, string(p))
+	assert.Equal(s.t, "/dp", r.URL.Path)
+}
+
+func TestLogsPusher(t *testing.T) {
+	suite.Run(t, &LogsPusherTestSuite{})
+}
+
+type LogsPusherTestSuite struct {
+	suite.Suite
+}
+
+func (s *LogsPusherTestSuite) TestLogsPusher() {
+	s.Run("someTest", func() {
```

Comment:
"someTest"?
I guess we don't need suite at this point and just one test should be enough